controllers/AuthController.js

```diff
@@ -162,9 +162,11 @@ export default class AuthController {
         type,
       });
 
-      return successResponse(res, 200, 'user', { message: 'You have successfully logged in', token });
-    } catch (error) {
       /* istanbul ignore next-line */
+      if (createdUser[0].isVerified === true) {
+        res.redirect(`${process.env.FRONTEND_URL}/signup?token=${token}`);
+      }
+    } catch (error) {
```

Comment:
Why are you doing this redirecting from the backend?

Reply:
initially, our socialLogin was just returning a user json object on the frontend, which I couldnt use to do anything. So what I did was return a frontend link containing the user's token as a query parameter, which would enable me to extract the token.

Reply:
From the `successResponse` you changed, the token is already being returned. Is that not the token you are looking to extract?

Reply:
Yes, but I didnt hsve any way I could return that `token` to the frontend except by redirecting it. So instead of returning a success response, I redirect it to the frontend, where everything else happens automatically.

Reply:
I'm sure you're making a request to the social login endpoint somewhere in your frontend code. If that is the case, then the token should be available in the response object